.travis.yml

```diff
@@ -112,7 +112,7 @@ before_install:
   - if [ "${USE_WHEEL}" == "1" ]; then pip install wheel; fi
   - |
     if [ "${REFGUIDE_CHECK}" == "1" ]; then
-        travis_retry pip install matplotlib Sphinx==1.2.3
+        travis_retry pip install matplotlib Sphinx==1.5.5
```

Comment:
Why do we force a specific version? Not doing so would ensure future version compatibility.